From: Oliver Neukum <oliver@neukum.org>
Date: Wed, 5 Oct 2011 12:16:03 +0200
Subject: [PATCH] USB: pwc: add dummy module parameter
Patch-mainline: never
References: bnc#720457

This adds back a module parameter which was available
but unfunctional with some kernel configs

Signed-off-by: Oliver Neukum <oneukum@suse.de>
---
 drivers/media/video/pwc/pwc-if.c |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)
--- a/drivers/media/video/pwc/pwc-if.c.alt	2011-10-05 12:23:03.000000000 +0200
+++ b/drivers/media/video/pwc/pwc-if.c	2011-10-05 12:24:47.000000000 +0200
@@ -1825,6 +1825,9 @@ module_param(fbufs, int, 0444);
 module_param(mbufs, int, 0444);
 #ifdef CONFIG_USB_PWC_DEBUG
 module_param_named(trace, pwc_trace, int, 0644);
+#else
+int dummy;
+module_param_named(trace, dummy, int, 0644);
 #endif
 module_param(power_save, int, 0444);
 module_param(compression, int, 0444);
@@ -1835,9 +1838,7 @@ MODULE_PARM_DESC(size, "Initial image si
 MODULE_PARM_DESC(fps, "Initial frames per second. Varies with model, useful range 5-30");
 MODULE_PARM_DESC(fbufs, "Number of internal frame buffers to reserve");
 MODULE_PARM_DESC(mbufs, "Number of external (mmap()ed) image buffers");
-#ifdef CONFIG_USB_PWC_DEBUG
 MODULE_PARM_DESC(trace, "For debugging purposes");
-#endif
 MODULE_PARM_DESC(power_save, "Turn power save feature in camera on or off");
 MODULE_PARM_DESC(compression, "Preferred compression quality. Range 0 (uncompressed) to 3 (high compression)");
 MODULE_PARM_DESC(leds, "LED on,off time in milliseconds");
